src/test/groovy/org/mars/rover/kata/CommandParserSpec.groovy

```diff
@@ -0,0 +1,30 @@
+package org.mars.rover.kata
+
+import org.mars.rover.kata.commands.Command
+import org.mars.rover.kata.commands.CommandParser
+import org.mars.rover.kata.commands.UnknownCommand
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class CommandParserSpec extends Specification {
```

Comment:
Bardzo dobrze się czyta te speckę, świetnie to napisałeś! Widać ewolucję od obsługi błędu po pozytywną ścieżkę. Szybko można zrozumieć, jakie są wymagania funkcjonalne do `CommandParser`a.